extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/AstroHandlerFactory.java

```diff
@@ -21,6 +20,7 @@
 import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
 import org.eclipse.smarthome.core.thing.binding.ThingHandler;
 
+import com.google.common.collect.Maps;
```

Comment:
As we are using Guava for preconditions already, I think it would do no harm to have such concise code.